2022-07_sprint/schedule.md

```diff
@@ -4,19 +4,17 @@
 :::{include} schedule_table.md
 :::
 
+
 :::{dropdown} Open dropdown for spreadsheet with times in more time zones
 :icon: clock
 
 <iframe src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSJ_x4V63aji-WmarsgEyj9eS_Cp0DB5m647TZdNjYj03nPQq0u0Q6eKPGhHrw-8Ugp16KhAg-AgLkK/pubhtml?widget=true&amp;headers=false"  height="320" width="750"></iframe>
+
 :::
 
 ## Schedule with Timezones
 
-<p float="left">
-   <a href="https://docs.google.com/spreadsheets/d/1oO9OhsTh9PFcqKI7Exj_ZcH0GvunQVHtM4aqUh0kjTk/edit?usp=sharing">
-<img alt="schedule with timezones" src="../_static/2022_07/2022_07_timezones.png">
-</a>
-</p>
+[![schedule with timezones](../_static/2022_07/2022_07_timezones.png)](https://docs.google.com/spreadsheets/d/1oO9OhsTh9PFcqKI7Exj_ZcH0GvunQVHtM4aqUh0kjTk/edit?usp=sharing)
```

Comment:
I think only the image or the embedding should be present, but not both. What do you prefer?

* dropdown vs no dropdown
* image vs embedding